src/main.ts

```diff
@@ -0,0 +1,25 @@
+import { fetchProjectItems } from "./fetchProjectItems";
+import { config } from "./config";
+import * as fs from "fs";
```

Comment:
Considering that you're `await`ing on it... shouldn't it be 
```ts
import { promises as fs } from "fs"
```
?

Reply:
[1970a31](https://github.com/paritytech/roadmap/pull/10/commits/1970a311259a87f2f52d8d330443b1fa13c289fe)